src/neural/network_opencl.cc

```diff
@@ -94,8 +94,8 @@ class OpenCLComputation : public NetworkComputation {
     policy_data_.emplace_back(move(policy_data));
 
     // Now get the score
-    double winrate =
-        Transforms::Innerproduct(weights_.ip2_val_w, value_data_) + weights_.ip2_val_b[0];
+    double winrate = Transforms::Innerproduct(weights_.ip2_val_w, value_data_) +
+                     weights_.ip2_val_b[0];
```

Comment:
If splitting here, then I'd put the `+` at the beginning of the second line

Reply:
clang-format won't agree. :)

Reply:
About the fact that thread=1 is better, but we still offer the parameters. I guess,
it enable users to check by themselves...